ONSdigital/eq-survey-runner
# eq 303, create mci-0215, write build time json, update python model, …

### Changes

Created 0215 and 0213 (update 0203, 0205 and star wars) to python model. Introduce build time json creator. Fix 
### How to test
1. Run tests
2. Open surveys 0215, 0213, 0203, 0205 and Star Wars in app  making sure the models are being generated and the front end works as expected
### Who can test

Anyone but @LJBabbage
